addons/project/models/res_partner.py

```diff
@@ -80,3 +80,30 @@ def action_view_tasks(self):
         else:
             action['domain'] = search_domain
         return action
+
+    @api.model
+    def get_mention_suggestions_from_task(self, task_id, search, limit=8):
```

Comment:
public model method not callable by guest, is that on purpose?

Reply:
The only use case of this method is to fetch mentions in the project sharing webclient, where the user is always logged in.